src/services/accommodation.service.js

```diff
@@ -0,0 +1,40 @@
+import models from '../database/models';
+
+const { booking, accommodation } = models;
+
+/**
+ * @description Trip requests service
+ */
+export default class RequestService {
+  /**
+   *@description Saves booking information in database
+   * @param {Object} bookingInfo trip request data
+   * @returns {Object} Saved booking information
+    */
+  static handleBookAccommodation(bookingInfo) {
```

Comment:
can you rename this function? `handleBookAccommodation` is not very descriptive, you can rename it to `saveBookedAccommodation`.